src/utils/formatValue.js

```diff
@@ -54,4 +54,12 @@ const formatValue = (formatter, value) => {
   }
 };
 
+export const toPercent = (decimal) => `${(decimal * 100).toFixed(0)}%`;
+
+export const getPercent = (value, total) => {
+  const ratio = total > 0 ? value / total : 0;
+
+  return toPercent(ratio, 2);
```

Comment:
Here we don't want to format anything, just want to do math and return the value. The caller should be responsible for the formatting.